superset/assets/src/preamble.js

```diff
@@ -0,0 +1,26 @@
+import 'abortcontroller-polyfill/dist/abortcontroller-polyfill-only';
+import { configure } from '@superset-ui/translation';
+import setupClient from './setup/setupClient';
+import setupColors from './setup/setupColors';
+
+// Configure translation
+if (typeof window !== 'undefined') {
+  const root = document.getElementById('app');
+  const bootstrapData = root ? JSON.parse(root.getAttribute('data-bootstrap')) : {};
+  if (bootstrapData.common && bootstrapData.common.language_pack) {
+    const languagePack = bootstrapData.common.language_pack;
+    delete bootstrapData.common.locale;
+    delete bootstrapData.common.language_pack;
```

Comment:
do these `delete` statements actually do anything since this is JSON copied/parsed from the HTML?

Reply:
I am confused by this one too. Probably ok to remove I guess. super weird

Reply:
removed.